content/c-sharp/concepts/strings/terms/split/split.md

```diff
@@ -0,0 +1,116 @@
+---
+Title: '.Split()'
+Description: 'Used to break string into substrings by specifying a delimeter.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Libraries'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The `.Split()`** method breaks a string into substrings based on a delimeter which can be a character or another string and returns an array of strings
+
+## Syntax
+
+```pseudo
+// Splits a string using a character as a delimeter
+string[] subs = String.Split(char)
+
+// Splits a string using a string as a delimeter
+string[] subs = String.Split(string)
+
+// Splits a string using a character as a delimeter into maximum substrings of the number specified in the second argument.
+string[] subs =  String.Split(char, Int)
+
+// Splits a string using a character as a delimeter into maximum substrings of the number specified in the second argument.
+string[] subs =  String.Split(string, Int)
```

Comment:
Can we separate these as follows:

Using a single character as a delimiter:
```pseudo
string[] subs = String.Split(char)
```

Using a string as a delimiter:
```pseudo
string[] subs = String.Split(string)
```


and others...